packages/blade/.storybook/react-native/storybook.requires.js

```diff
@@ -1,6 +1,29 @@
 /* do not change this file, it is auto generated by storybook. */
 
-import { configure, addDecorator, addParameters, addArgsEnhancer } from '@storybook/react-native';
+import {
+  configure,
+  addDecorator,
+  addParameters,
+  addArgsEnhancer,
+  clearDecorators,
+} from '@storybook/react-native';
+
+global.STORIES = [
+  {
+    titlePrefix: '',
+    directory: './src',
+    files: '**/*.stories.?(ts|tsx|js|jsx)',
+    importPathMatcher:
+      '^\\.[\\\\/](?:src(?:\\/(?!\\.)(?:(?:(?!(?:^|\\/)\\.).)*?)\\/|\\/|$)(?!\\.)(?=.)[^/]*?\\.stories\\.(?:ts|tsx|js|jsx)?)$',
```

Comment:
damn thats a fat regex, what does this code do?

Reply:
Reviewing the wrong PR @anuraghazra 

You need to review the other upgrade PR 😆 #1507

Reply:
lol okay, i'm done with this PR anyway.